src/java.desktop/unix/native/libawt_xawt/xawt/XToolkit.c

```diff
@@ -692,7 +692,12 @@ void awt_output_flush() {
 static void wakeUp() {
     static char wakeUp_char = 'p';
     if (!isMainThread() && awt_pipe_inited) {
-        write ( AWT_WRITEPIPE, &wakeUp_char, 1 );
+        if (write(AWT_WRITEPIPE, &wakeUp_char, 1) < 0) {
+            // if block is left empty to avoid adding unused-result to
```

Comment:
Does it make sense to add a trace with `DTRACE_PRINTLN*`?

It could be useful for debugging…

Reply:
yes If write does not return 1 might indicate an error, it would be good to log it. so we can skip this long comment

Reply:
Updated